google/cloud/storage/benchmarks/storage_latency_benchmark.cc

```diff
@@ -130,11 +130,17 @@ int main(int argc, char* argv[]) try {
     return 1;
   }
 
-  gcs::ClientOptions client_options;
+  google::cloud::StatusOr<gcs::ClientOptions> client_options =
+      gcs::ClientOptions::CreateDefaultClientOptions();
+  if (!client_options.ok()) {
+    std::cerr << "Could not create ClientOptions, status="
+              << client_options.status() << std::endl;
+    return 1;
+  }
   if (!options.enable_connection_pool) {
-    client_options.set_connection_pool_size(0);
+    client_options->set_connection_pool_size(0);
   }
-  gcs::Client client(client_options);
+  gcs::Client client(*client_options);
```

Comment:
Consider: `gcs::Client client(*std::move(client_options));` I do not think it matters in terms of performance, but maybe we want to show good practices in general?

Reply:
Done.